Firestore/Source/Remote/FSTRemoteStore.mm

```diff
@@ -145,22 +153,18 @@ - (void)setOnlineStateDelegate:(nullable id<FSTOnlineStateDelegate>)delegate {
 #pragma mark Online/Offline state
 
 - (BOOL)isNetworkEnabled {
```

Comment:
This got renamed to canUseNetwork, with a porting note: https://github.com/firebase/firebase-android-sdk/blob/98326c431a189bc84829ad0c7df0089202f57990/firebase-firestore/src/main/java/com/google/firebase/firestore/remote/RemoteStore.java#L447